test/server/admin/admin_test.cc

```diff
@@ -248,6 +489,91 @@ TEST_P(AdminInstanceTest, ConfigDumpFiltersByResource) {
   EXPECT_EQ(expected_json, output);
 }
 
+TEST_P(AdminInstanceTest, ConfigDumpWithEndpointFiltersByResource) {
```

Comment:
Please add a single line `//` above each test case to explain the intuition of what it is validating.

Reply:
Added